docs/source/bird-rr-config.rst

```diff
@@ -0,0 +1,149 @@
+Configuring BIRD as a BGP Route Reflector
+=========================================
+
+These instructions will take you through installing BIRD as a BGP route
+reflector, and updating your other BIRD instances to speak to your new route
+reflector.
+
+These instructions are valid for both Ubuntu 14.04 and RHEL 7.
+
+Prerequisites
+-------------
+
+Before starting this you will need the following:
+
+- A machine running either Ubuntu 14.04 or RHEL 7.
```

Comment:
Whilst it should be obvious, I think it would be worth noting this is a "fresh" machine, which will become your route reflector (i.e. not just a random machine you've got lying around that, perhaps, is already acting as a compute node...).